README.md

```diff
@@ -579,28 +564,30 @@ let {
 Object.defineProperty(C.prototype, "x", { get: newGet, set: newSet });
 ```
 
-### `@init:` Decorators
+### Adding initialization logic with `addInitializer`
 
-The `@init:` syntax can be used with any decorator, and adds the `addInitializer` method to the context object that is provided to the decorator. This method can be called to associate an initializer function with the class or class element, which can be used to run arbitrary code after the value has been defined in order to finish setting it up. The timing of these initializers depends on the type of decorator:
+The `addInitializer` method is available on the context object that is provided to the decorator. This method can be called to associate an initializer function with the class or class element, which can be used to run arbitrary code after the value has been defined in order to finish setting it up. The timing of these initializers depends on the type of decorator:
 
-- Class decorator initializers are run after the class has been fully defined, and class static fields have been assigned.
-- Class element initializers run during class construction, and are interleaved with standard class field initializers. Like class field initializers, they run in element order, and after any pre-existing standard initializer (e.g. the initializer for a class field or auto-accessor).
-- Class _static_ element initializers run during class definition, and are interleaved with standard static class field initializers, similar to non-static class element initializers.
+- Class decorator initializers are run _after_ the class has been fully defined, and _after_ class static fields have been assigned.
+- Class method and accessor initializers run during class construction, _before_ class fields are initialized.
+- Class field initializers run after the field has been defined on the class instance.
+- Class _static_ method and element initializers run during class definition, _before_ static class fields are defined, but _after_ class elements have been defined.
+- Class _static_ field initializers run after the field has been defined on the class.
 
-#### Example: `@init:customElement`
+#### Example: `@customElement`
 
-We can use `@init:` with class decorators in order to create a decorator which registers a web component in the browser.
+We can use `addInitializer` with class decorators in order to create a decorator which registers a web component in the browser.
```

Comment:
This change makes me very happy. I think the model is simpler and it also has backwards compatibility with the current legacy consumer syntax.